integration-tests/README.md

```diff
@@ -87,10 +87,10 @@ Run the tests with:
 
 ## Testing an update
 
-With the --update flag you can flash an old image, update to the latest and
-then run the whole suite on the updated system. The release, the channel and
-the revision flags specify the image that will be flashed, and the
-target-release and target-channel flags specify the values to be used in the
+With the `--update` flag you can flash an old image, update to the latest and
+then run the whole suite on the updated system. The `release`, the `channel` and
+the `revision` flags specify the image that will be flashed, and the
+`target-release` and `target-channel` flags specify the values to be used in the
 update if they are different from the flashed values.
 
 For example, to update from rolling edge -1 to the latest and then run the
```

Comment:
mmm, good point. I pushed a new version that uses *...* for values and `...` for flags and vars.